backend/eventloop/application_event_loop/application_event_runner_deployments.go

```
@@ -661,6 +664,13 @@ func (a *applicationEventLoopRunner_Action) applicationEventRunner_handleUpdateD
 	gitopsDeployment.Status.Sync.Status = managedgitopsv1alpha1.SyncStatusCode(applicationState.Sync_Status)
 	gitopsDeployment.Status.Sync.Revision = applicationState.Revision
 
+	// Fetch the list of resources created by deployment from table and update local gitopsDeployment instance.
+	var err error
+	gitopsDeployment.Status.Resources, err = decompressResourceData(applicationState.Resources)
+	if err != nil {
+		a.log.Error(err, "unable to decompress byte array received from table.")
+	}
```

Should return the error if it is non-nil